tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/doNotExitDockerContainer.ts

```diff
@@ -0,0 +1,6 @@
+import * as fs from "fs";
+
+export function doNotExitDockerContainer() {
+    // this file will be used by entrypoint.sh to determine whether exit the docker container.
+    fs.writeFileSync('/tmp/notExit', 'yes', 'utf-8');
```

Comment:
do we have scenario  delete this file when we don't need task engine docker?

Reply:
No. This file only exists when docker is running in local, no matter running task engine or not.